### PR TITLE
Issue259 allow epsg code as string

### PR DESCRIPTION
**Pull request by JohanKJSchreurs**
Implements https://github.com/Open-EO/openeo-python-client/issues/259

### CODE REVIEW

File: openeo/util.py [hunk 1]
```diff
@@ -627,3 +630,74 @@ def get(self, fraction: float) -> str:
         width = self.width - len(self.left) - len(self.right)
         bar = self.bar * int(round(width * clip(fraction, min=0, max=1)))
         return f"{self.left}{bar:{self.fill}<{width}s}{self.right}"
+
+
+class EPSGCodeNotFound(Exception):
```
Comment (soxofaan) [inline openeo/util.py:635]: "not found" seems like a strange expressions for a CRS code, maybe make it "invalid EPSG code". Or just keep it to a vanilla `ValueError` with a good message?

Comment (JohanKJSchreurs) [reply openeo/util.py]: This was indeed not ideal, I was in fact already planning to remove that one.
Now these are ValueErrors. (And typerrors for anything that is not str, int or None)

**Comment by jdries** [comment]
fine for me!

**Comment by JohanKJSchreurs** [comment]
Have found out that we should also support PROJJSON.  Adding some test coverage for that as well.

See also:
https://github.com/Open-EO/openeo-processes/issues/58
https://github.com/Open-EO/openeo-processes/commit/2684dab980b02bad8fe7aa2d59049462ca382760

**Pull request merged**